server.py

```diff
@@ -1,296 +0,0 @@
-
-        # if user does not select file, browser also
-        # submit a empty part without filename
-        if file.filename == '':
-            # flash('No selected file')
-            return redirect("/")
-
-        if file and allowed_file(file.filename):
-            filename = secure_filename(file.filename)
-            
-            # Save file to disk
-            path_to_save_file = os.path.join(app.config['UPLOAD_FOLDER'], 
-                filename)
-            file.save(path_to_save_file)
-
-            # Save to DB
-            glitz_id = save_glitz_data(path=path_to_save_file, 
-                                    user_id=session["user_id"], title='yeah!')
-
-            return redirect("/view_glitz/" + str(glitz_id))
-            # return render_templatew_glitz.html", file_path=path_to_save_file)
-
-    return redirect("/")
-
-
-# Make new route and new def for that route
-@app.route('/view_glitz/<int:glitz_id>')
-def view_glitz(glitz_id):
-    
-    if not is_user_logged_in(session):
-        return redirect("/login")
-
-    # Using glitz_id, we need to fetch glitz data from DB
-    
-    grids = Grid.query.all()
-
-    glitz_feeds = (db.session.query
-                                (Glitz.glitz_path
-                                    ,Glitz.glitz_id
-                                    ,Comment.comment_text
-                                    ,User.user_id
-                                    ,(User.fname + ' ' + User.lname).label('name')
-                                    ,Glitz.posted_on)
-                                .outerjoin(Comment)
-                                .outerjoin(User)
-                                .filter(Glitz.glitz_id == glitz_id)
-                                .all())
-
-    glitz_comments = get_glitz_comments(glitz_feeds)  
-
-    return render_template("view_glitz.html", glitz_comments=glitz_comments
-                            ,grids=grids)                                                   
-
-
-
-    # return render_template("view_glitz.html", 
-    #                         file_path="../" + glitz.glitz_path,
-    #                         grids=grids,
-    #                         glitz_id=glitz_id)
-
-
-@app.route('/save_comment', methods=['POST'])
-def save_comment():
-
-    if not is_user_logged_in(session):
-        return redirect("/login")
-    
-    # Get user_id from session 
-    user_id = session["user_id"]
-    
-    # Get form variables
-    comment = request.form["comment"]
-    grid_id = request.form["select_grids"]
-    glitz_id = request.form["glitz_id"]
-
-    # save the comment on db
-    comment_id = save_comment_data(glitz_id, user_id, grid_id, comment)
-
-    # We need to pass in glitz_id, grid_id and comment_text from HTML form
-    return redirect("/view_glitz/" + glitz_id)
-
-
-@app.route('/view_comments/<int:glitz_id>', methods=['GET', 'POST'])
-def view_comments(glitz_id):
-
-    if not is_user_logged_in(session):
```

Comment:
You can use the @login_required decorator to make it so you don't have to add this to every call.
http://flask.pocoo.org/docs/0.12/patterns/viewdecorators/#login-required-decorator